src/sentry/search/events/filter.py

```diff
@@ -3,7 +3,7 @@
 from collections.abc import Callable, Mapping, Sequence
```

Comment:
this file is ignored which is why this wasn't caught before -- will follow up to hopefully get this file unignored